content/enterprise/requirements/data-storage/vault.mdx

```diff
@@ -39,45 +39,71 @@ Terraform Enterprise.
 1. Create the `tfe-policy.hcl` file with the following content:
 
   ```hcl
-  path "auth/approle/login" {
-    capabilities = ["create", "read"]
+  # To renew leases.
+  path "sys/leases/renew/*" {
+    capabilities = ["create", "read", "update"]
   }
```

Comment:
This path supersedes sys/renew. [Source](https://github.com/hashicorp/vault/blob/main/CHANGELOG.md#080-august-9th-2017).